packages/victory-core/src/victory-util/textsize.ts

```diff
@@ -316,11 +324,14 @@ const _measureDimensionsInternal = (
   containerElement.innerHTML = "";
 
   return { width, height };
-};
+}, (text, style) => {
+  const totalText = Array.isArray(text) ? text.join() : text;
```

Comment:
This is cool! 
Maybe you could really simplify by doing `JSON.stringify` here, since it's ultra optimized.

Reply:
I chose not to go for json.stringify simply because I wanted to make sure the attributes were in a consistent order, but that may or may not actually be a concern we're worried about. Happy to change it if we feel it's not a big deal.